src/cmd/support_archive/mocks/pod_interface.go

```diff
@@ -0,0 +1,855 @@
+// Code generated by mockery v2.15.0. DO NOT EDIT.
+
+package mocks
```

Comment:
this is cool, but I dunno if we will use mockery (its veeeeery likely)

couldn't this be in a more general location ?
so it can be reused by other packages if needed
- `PodInterface` from `k8s.io/client-go/kubernetes/typed/core/v1` sounds something that might be useful somewhere else as well 😄

Reply:
For this mock it isn't even necessary, that we use mockery as a general tool. I just used it to safe some tedious work of coding a mock manually and I had at hand (coincidently 😛).

But you are right, the location is not optimal.